requirements.txt

```diff
@@ -1,10 +1,11 @@
 Flask
 Flask-Cache
-cairocffi
-pyparsing>=1.5.7
+cairocffi<0.7
+pyparsing>=2.1.10
```

Comment:
I put pyparsing>=2.0 because of this issue: 
https://github.com/pypa/packaging/issues/94
trying to recover cairocffi without version hardcoded but looks like it's failing, please check last build log